simexpal/queuesock.py

```diff
@@ -50,6 +63,13 @@ def run(self):
 					connection = Connection(conn)
 
 					self.selector.register(conn, selectors.EVENT_READ, connection)
+				elif sk.fileobj == signal_reader:
+					if cur_subproc is not None:
+						cur_subproc.terminate()
```

Comment:
Why do we use terminate() and not kill() here (as in launchers/common.py)?

Reply:
The subprocesses in ``queuesock.py`` are all simexpal processes that terminate accordingly when receiving ``SIGTERM`` or ``SIGINT``.
The subprocesses in ``common.py`` can be arbitrary processes, which could catch ``SIGTERM`` and ``SIGINT`` and keep runing. Thus, we use ``kill()`` in ``common.py``, which sends ``SIGKILL`` to make sure, we terminate the subprocess.